tests/common.go

```diff
@@ -7398,3 +7457,97 @@ func deleteNamespaces(namespaces []string) error {
 	}
 	return nil
 }
+
+// CreateCloudCredentialNew creates cloud credentials
+func CreateCloudCredentialNew(provider string, ctx context1.Context, orgID string, credName string, uuid string, kubeconfig string) error {
+	log.Infof("Create cloud credential with name [%s] for org [%s] with [%s] as provider", credName, orgID, provider)
+	var credCreateRequest *api.CloudCredentialCreateRequest
+	switch provider {
+	case drivers.ProviderAws:
+		log.Infof("Create creds for Aws")
+		// PA-1328
+		id := os.Getenv("AWS_ACCESS_KEY_ID")
+		if id == "" {
+			return fmt.Errorf("environment variable AWS_ACCESS_KEY_ID should not be empty")
+		}
+		// PA-1328
+		secret := os.Getenv("AWS_SECRET_ACCESS_KEY")
+		if secret == "" {
+			return fmt.Errorf("environment variable AWS_SECRET_ACCESS_KEY should not be empty")
+		}
```

Comment:
How about s3utils.GetAWSDetailsFromEnv()?

Reply:
As discussed , it will be removed once PR for PA-1328 is mered